README.md

```diff
@@ -1,2 +1,25 @@
 # sgl-learning-materials
+
 Materials for learning SGLang
+
+## Blog
+
+[2024-09-04] [SGLang v0.3 Release: 7x Faster DeepSeek MLA, 1.5x Faster torch.compile, Multi-Image/Video LLaVA-OneVision](https://lmsys.org/blog/2024-09-04-sglang-v0-3/)
+
+[2024-08-15] [Supercharge Your LLMs Using SGLang For Inference: Why Speed and Efficiency Matter More Than Ever](https://blog.runpod.io/supercharge-your-llms-using-sglang/)
```

Comment:
Remove this as it does not meet our blog's quality standard.

Reply:
ok